src/nvim/getchar.c

```diff
@@ -1655,10 +1655,6 @@ static void getchar_common(typval_T *argvars, typval_T *rettv)
   no_mapping++;
   allow_keys++;
   for (;;) {
-    // Position the cursor.  Needed after a message that ends in a space,
-    // or if event processing caused a redraw.
-    ui_cursor_goto(msg_row, msg_col);
```

Comment:
I think this breaks backward-compatibility.

Reply:
fixed.